### ISSUE TITLE
Not a valid aggregate or forensic report

### ISSUE THREAD

**Issue by xennn** [domainaware/parsedmarc#18]
Cant Parse Some Reports:
Mail:
Report Domain: xxxxxx.de Submitter: ikea.com Report-ID: 2018_10_05_5bb7e9b4f3e8a
This is an aggregate report from ikea.com.

Attachment:
ikea.com!xxxxxx.de!1538690400!1538776800.xml.gz
ERROR:parsedmarc:Failed to parse ikea.com!xxxxxx.de!1538690400!1538776800.xml - Not a valid aggregate or forensic report
{
  "aggregate_reports": [],
  "forensic_reports": []
}

Example:
https://pastebin.com/Pzgq4eGh

**Comment by seanthegeek** [comment]
Thanks for the report. The sample from IKEA is not valid XML, because of this tag

```xml
<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" targetNamespace="http://dmarc.org/dmarc-xml/0.1">
```

Without a corresponding `</xs:schema>` closing tag.

These tags are used for XML Schema Definition (XSD) files do not belong in a XML file.

That said, I just released `parsedmarc` 4.1.7 to work around this problem by removing any invalid XML schema tags before parsing the XML.

I'll also reach out to IKEA to let them know they are generating invalid XML.

**Comment by xennn** [comment]
No Problem. Great Thank you for the fast fix! Yes this is a good idea to give IKEA a note.
